programs/uxd/src/utils/calculate_depositories_target_redeemable_amount.rs

```diff
@@ -224,32 +229,17 @@ fn calculate_depository_target_amount(
             .checked_sub(depository_available_amount)
             .ok_or(UxdError::MathError)?;
         compute_amount_less_fraction_floor(
-            total_overflow_amount,
+            std::cmp::min(total_overflow_amount, total_available_amount),
```

Comment:
this is a bugfix surfaced by more advanced prop test

Reply:
Nice, can add a comment for the bug maybe